packages/template-retail-react-app/app/pages/checkout/util/checkout-context.js

```diff
@@ -186,20 +186,11 @@ export const CheckoutProvider = ({children}) => {
 
             // Async functions
             // Convenience methods for interacting with remote customer and basket data.
+            //
+            // @TODO: ALL METHODS BELOW SHOULD BE REMOVED BY THE END OF HOOK INTEGRATION
```

Comment:
Are we planning to keep getters in this provider?

Reply:
My goal is to remove as much data fetching related properties/functions. Keep the logics that manages the checkout steps.

Reply:
I want to chip away this context piece by piece in the upcoming tickets. Not sure if you are referring to any actionable changes in this PR?